Transaction pool test, TestTicksOnSchedule flaky

was running in local docker, got this output

--- FAIL: TestTicksOnSchedule (0.01s)
	cleaner_test.go:38: did not call cleaner within expected timeframe
FAIL
FAIL	github.com/orbs-network/orbs-network-go/services/transactionpool	0.017s

--- a/services/transactionpool/cleaner_test.go
+++ b/services/transactionpool/cleaner_test.go
@@ -7,7 +7,7 @@ import (
 	"time"
 )
 
-var tickInterval = func() time.Duration { return 100 * time.Microsecond }
+var tickInterval = func() time.Duration { return 1 * time.Millisecond }
 var expiration = func() time.Duration { return 30 * time.Minute }
 
 func TestStopsWhenContextIsCancelled(t *testing.T) {

